.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: false
 
 before_install: 
 
-- apt-get install libeigen3-dev
+- sudo apt-get install libeigen3-dev
```

Comment:
You can't use `sudo` on the `sudo: false` machines on Travis. Try doing this to install dependencies:
```
addons:
  apt:
    packages:
    - cmake
    - time
```